l10n_ar_wsfe/wsfetools/wsfe_easywsy.py

```diff
@@ -56,11 +56,9 @@ def parse_invoice(self, invoice, number=False, invoices=False):
         if not number:
             number = invoice.split_number()[1]
 
-        date_invoice = datetime.strptime(invoice.date_invoice, '%Y-%m-%d')
+        date_invoice = invoice.date_invoice
         formatted_date_invoice = date_invoice.strftime('%Y%m%d')
```

Comment:
`date_invoice` here isn't a string? I think you need to use `fields.Date.from_string()` to cast that string to a datetime object

Reply:
invoice.date_invoice is a datetime.date()